pkg/kubelet/network/hostport/hostport_manager.go

```diff
@@ -53,14 +52,23 @@ type hostportManager struct {
 
 func NewHostportManager() HostPortManager {
 	iptInterface := utiliptables.New(utilexec.New(), utildbus.New(), utiliptables.ProtocolIpv4)
+
+	// Ensure the chain is linked from the root.
+	// FIXME: This is a hack. It will update as new pods come in, but we really
+	// should periodically resync this.
+	if err := ensureKubeHostportChainLinked(iptInterface); err != nil {
+		glog.Errorf("Failed to ensure hostport chains are linked, will retry: %v", err)
```

Comment:
"...will retry on next pod creation"

Probably should be explicit so people don't go looking for code that doesn't exist